extensions/packageManager/src/client/Library.tsx

```diff
@@ -391,15 +537,15 @@ const Library: React.FC = () => {
           </Link>
         </p>
       </div>
-      <Stack horizontal disableShrink styles={{ root: { borderTop: '1px solid #CCC' } }}>
-        <Stack.Item styles={{ root: { width: '300px', borderRight: '1px solid #CCC' } }}>
+      <Stack horizontal styles={{ root: { borderTop: '1px solid #CCC' } }}>
+        {/* <Stack.Item styles={{ root: { width: '300px', borderRight: '1px solid #CCC' } }}>
```

Comment:
remove commented code

Reply:
this actually needed to be turned back on.